Report/Report.cs

```diff
@@ -109,6 +110,12 @@ public Report(string name, string description, string version, BacktestResult ba
                 }
             }
 
+            // Activate virtual environment if defined
+            PythonInitializer.ActivatePythonVirtualEnvironment(pythonVirtualEnvironment);
+
+            // Initialize and add our Paths
+            PythonInitializer.Initialize();
```

Comment:
Hey @AibekMinbaev! Thanks for the PR, sorry let's move these to `Report/Program.cs` 👍